release/generate_changelog.py

```diff
@@ -0,0 +1,38 @@
+"""Generate changelog entries for all PRs merged since the last release."""
+import requests
+
+get = requests.get('https://api.github.com/repos/RaRe-Technologies/gensim/releases')
+get.raise_for_status()
+
+last_release_date = get.json()[0]['published_at'][:10]
+
+
+def g():
+    page = 1
+    done = False
+
+    while not done:
+        get = requests.get(
+            'https://api.github.com/repos/RaRe-Technologies/gensim/pulls',
+            params={'state': 'closed', 'page': page},
```

Comment:
Is this guaranteed to be sorted by `created_at`?

Reply:
I can't find any guarantees anywhere, but in practice, yes, it seems to be.